resources/public/api/conf/2.0/application.raml

```diff
@@ -655,7 +653,30 @@ traits:
                          "arrivalId": "62f4ebbbf581d4aa",
                          "received": "2022-08-11T11:44:59.83705",
                          "type": "IE007",
-                         "body": "<ncts:IE007 PhaseID=\"NCTS5.0\" xmlns:ncts=\"http://ncts.dgtaxud.ec\">...</ncts:IE007>"
+                         "body": "<ncts:CC007C PhaseID=\"NCTS5.0\" xmlns:ncts=\"http://ncts.dgtaxud.ec\">...</ncts:CC007C>"
+                       }
+                   application/json:
+                     description: JSON payload
+                     example: |
+                       {
+                         "_links":{
+                           "self":{
+                             "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa/messages/62f4ebbb765ba8c2"
+                           },
+                           "arrival":{
+                             "href": "/customs/transits/movements/arrivals/62f4ebbbf581d4aa"
+                           }
+                         },
+                         "id": "62f4ebbb765ba8c2",
+                         "arrivalId": "62f4ebbbf581d4aa",
+                         "received": "2022-08-11T11:44:59.83705",
+                         "type": "IE007",
```

Comment:
Same here: https://github.com/hmrc/common-transit-convention-traders/blob/3991baefbe9aa83be97973a9cdc6c58335a6e74e/app/models/response/HateoasArrivalResponseMessage.scala#L40

Reply:
Liandra has modified it and it is now using HateoasMovementMessageResponse and there it is using  "type"                            -> messageSummary.messageType
.

Reply:
Yup!